site/pkg/build/.coverprofile

```diff
@@ -0,0 +1,89 @@
+mode: atomic
+github.com/kedgeproject/kedge/pkg/build/build.go:43.79,46.16 2 0
```

Comment:
kedge :question:

Reply:
good catch.
This file shouldn't be there. It is unit test coverage

Reply:
wow that's an old file. thanks! removed.